rebar.config

```diff
@@ -14,16 +14,16 @@
                ]}.
 
 {deps, [
-        {node_package, ".*", {git, "git://github.com/basho/node_package.git", {branch, "develop"}}},
-        {lager_syslog, "2.0.3", {git, "git://github.com/basho/lager_syslog.git", {tag, "2.0.3"}}},
-        {cluster_info, ".*", {git, "git://github.com/basho/cluster_info.git", {branch, "develop"}}},
-        {riak_kv, ".*", {git, "git://github.com/basho/riak_kv.git", {branch, "develop"}}},
-        {riak_search, ".*", {git, "git://github.com/basho/riak_search.git", {branch, "develop"}}},
-        {riak_control, ".*", {git, "git://github.com/basho/riak_control.git", {branch, "develop"}}},
+        {node_package, ".*", {git, "git://github.com/basho/node_package.git", {tag, "2.0.0"}}},
```

Comment:
Node package tag 2.0.0 doesn't actually contain the tip of the 2.0 branch that includes many new changes. We need to to create a new tag for the `node_package` repo on the 2.0 branch head. Then of course we'll have to ensure that changes get merged back to develop eventually. We also need to do the same thing for  any other repo that has commits later than the 2.0.0 tag. 

We then seriously need to consider our branching strategy on these repos. They don't have anywhere near the frequency of changes, or the formal release structure of riak/riak_ee/riak_core/riak_kv.

Usually jumping major and minor versions means significant change, but in our case it mainly means "matches riak/ee versioning." Since these versions change infrequently I suggest not developing on long lived versioned branches as much as we do now. For most changes we should probably just create a feature or bugfix branch directly off develop. Then we can simply merge that in and tag develop as part of the regular development process, updating rebar.config of parent projects as necessary. This will prevent the need to 'stabilize' a release and figure out "what is in it" for all repos. It also means we don't have to re-branch/re-tag each subrepo for each major release. 

Of course for repos with major changes and large, long running and diverging features we can create versioned branches as necessary.